File: raspi/blink.c
int pin = 11;
int lightTime = 1000;
int dimTime = 200;

using GpioController controller = new();
controller.OpenPin(pin, PinMode.Output);

while (true)
{
    controller.Write(pin, PinValue.High);
    Thread.Sleep(lightTime);
    controller.Write(pin, PinValue.Low);
    Thread.Sleep(dimTime);
}

File: raspi/blink.py
import RPi.GPIO as IO         # calling header file for GPIO’s of PI
import time                   # calling for time to provide delays in program

duration=1.5
IO.setmode (IO.BOARD)        # BOARD pin numbers, GPIO21 is called as PIN40

IO.setup(40,IO.OUT)          # initialize digital pin40 as an output.
IO.output(40,1)              # turn the LED on (making the voltage level HIGH)
time.sleep(duration)         # sleep for a second
IO.cleanup()                 # turn the LED off (making all the output pins LOW)
time.sleep(1)                #sleep for a second    

#loop is executed second time        
IO.setmode (IO.BOARD)
IO.setup(40,IO.OUT)
IO.output(40,1)
time.sleep(duration)
IO.cleanup()
time.sleep(1)

#loop is executed third time
IO.setmode (IO.BOARD)
IO.setup(40,IO.OUT)
IO.output(40,1)
time.sleep(duration)
IO.cleanup()
time.sleep(1)

